scripts/restapicli.sh

```
@@ -0,0 +1,198 @@
+            ;;
+        unban)
+            if [[ $# -lt 1 ]]; then
+                ee ">>> Missing userid argument for 'unban'"
+                print_usage
+                exit 1
+            fi
+            local userid=$1
+            restapi_post "unban" "{\"userid\":\"${userid}\"}" || exit 1
+            es "> Unbanned: ${userid}"
+            ;;
+        kick)
```

Shellcheck (0.11.0)

[warning] This pattern never matches because of a previous pattern on line 97.

(SC2222)